src/gtk3.mk

```diff
@@ -20,22 +20,37 @@ define $(PKG)_UPDATE
 endef
 
 define $(PKG)_BUILD
-    cd '$(BUILD_DIR)' && '$(SOURCE_DIR)/configure' \
-        $(MXE_CONFIGURE_OPTS) \
-        --disable-glibtest \
-        --disable-cups \
-        --disable-test-print-backend \
-        --disable-gtk-doc \
-        --disable-man \
-        --with-included-immodules \
-        --enable-win32-backend
-    $(MAKE) -C '$(BUILD_DIR)' -j '$(JOBS)' $(MXE_DISABLE_CRUFT) EXTRA_DIST=
-    $(MAKE) -C '$(BUILD_DIR)' -j 1 install $(MXE_DISABLE_CRUFT) EXTRA_DIST=
+    # Meson configure, with additional options for GTK
+    '$(MXE_MESON_WRAPPER)' $(MXE_MESON_OPTS) \
+        -Dtests=false \
+        -Dexamples=false \
+        -Ddemos=false \
+        -Dinstalled_tests=false \
+        -Dbuiltin_immodules=yes \
+        -Dintrospection=false \
+        '$(BUILD_DIR)' '$(SOURCE_DIR)' && \
+    '$(MXE_NINJA)' -C '$(BUILD_DIR)' -j '$(JOBS)' && \
+    DESTDIR="/" \
+        '$(MXE_NINJA)' -C '$(BUILD_DIR)' -j '$(JOBS)' install
+
+    #cd '$(BUILD_DIR)' && '$(SOURCE_DIR)/configure' \
+    #    $(MXE_CONFIGURE_OPTS) \
+    #    --disable-glibtest \
+    #    --disable-cups \
+    #    --disable-test-print-backend \
+    #    --disable-gtk-doc \
+    #    --disable-man \
+    #    --with-included-immodules \
+    #    --enable-win32-backend
+    #$(MAKE) -C '$(BUILD_DIR)' -j '$(JOBS)' $(MXE_DISABLE_CRUFT) EXTRA_DIST=
+    #$(MAKE) -C '$(BUILD_DIR)' -j 1 install $(MXE_DISABLE_CRUFT) EXTRA_DIST=
 
     # cleanup to avoid gtk2/3 conflicts (EXTRA_DIST doesn't exclude it)
     # and *.def files aren't really relevant for MXE
+    # TODO: Still relevant to remove?
     rm -f '$(PREFIX)/$(TARGET)/lib/gailutil.def'
```

Comment:
Is GTK3 still causing conflicts with this file with GTK2?

Reply:
Looking at `usr/TARGET/lib` folder - nope.

Reply:
Indeed. I will remove this workaround. So this file doesn't get removed (well it's not present to begin with).